packages/gateway/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "OpenCRVS API Gateway with GraphQL",
   "license": "MPL-2.0",
   "scripts": {
-    "start": "concurrently \"nodemon --exec ts-node -r tsconfig-paths/register src/index.ts\" \"yarn gen:types:watch\"",
+    "start": "cross-env LANGUAGES='en' concurrently \"nodemon --exec ts-node -r tsconfig-paths/register src/index.ts\" \"yarn gen:types:watch\"",
```

Comment:
if we hard code the LANGUAGES to 'en' here then it will have some impact on BGD dev env. Can we think of passing it from root package.json as param ? Default can be english there but if needed (like for BGD) we can pass any other value while starting the dev env.

Reply:
Yes, thats much better